srv/components/misc_pkgs/mocks/cortx/files/setup.mock.yaml/setup.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Can we use '-' in directory name instead of '.'?
'.' notation at times could be misleading.

Reply:
I agree. It was just as initial design structure. I fixed that